### ISSUE TITLE
pkgdown build fails

### ISSUE THREAD

**Issue by jforbes14** [ropenscilabs/eechidna#37]
Hey @bryceroney could you check out the [logs](https://github.com/jforbes14/eechidna/runs/1933283346?check_suite_focus=true) for the gh-pages pkgdown build? Build seems to be failing.

**Comment by bryceroney** [comment]
Yeah, I just saw. I wonder if the data files etc are reaching whatever
memory limit there are on GitHub actions.

I'll take a look over the weekend

**Comment by jforbes14** [comment]
Interestingly now get this error when I try to run `pkgdown::build_site()` 

@bryceroney when you get a chance, can you see if this error replicates when you run the above? Tried building site with previous commits and run into the same problem.

> pkgdown::build_site()
-- Installing package into temporary library ----------------------------------------------------------------------------
== Building pkgdown site =======================================================
Reading from: '/Users/Jeremy/Documents/R/eechidna'
Writing to:   '/Users/Jeremy/Documents/R/eechidna/docs'
-- Initialising site -----------------------------------------------------------
-- Building home ---------------------------------------------------------------
Reading 'CONDUCT.md'
Reading 'CONTRIBUTING.md'
-- Building function reference -------------------------------------------------
Reading 'man/aec_add_carto_f.Rd'
Error : Failed to parse Rd in aec_add_carto_f.Rd
ℹ unused argument (bg = bg)
Error: callr subprocess failed: Failed to parse Rd in aec_add_carto_f.Rd
ℹ unused argument (bg = bg)

**Comment by bryceroney** [comment]
I can't seem to replicate that one, have you updated all your R packages?

**Comment by jforbes14** [comment]
That and adding `\dontrun` to some of the examples for `nat_map` objects seemed to do the trick. Thanks!

> I can't seem to replicate that one, have you updated all your R packages?

**Comment by dicook** [comment]
@jforbes14 For the web site to work again:

1. Change the setting to build the web site from master, using the docs folder (this might already be the case because the page is deploying.)
2. Remove the GitHub actions that build to gh-pages
3. Add link to the web site in the About section of the main repo page, https://jforbes14.github.io/eechidna/

**Comment by dicook** [comment]
Oh, image of hex sticker should be added to web page

**Comment by jforbes14** [comment]
Done 1 & 3. Looks like the page is building now (thanks to some work by @dicook or @robjhyndman ?) and I'm not seeing any GitHub actions that build to gh-pages. Am I missing something or is this issue complete @dicook ?

**Comment by dicook** [comment]
On 3. The About block is on the right at the top. This is the place to add the URL.

**Comment by dicook** [comment]
I just removed the pkgdown.yml in the .github/workflows folder which should fix 2

**Comment by dicook** [comment]
You might need to go into the Actions tab @jforbes14 to remove the pkgdown workflow

**Comment by jforbes14** [comment]
Ah great - About is updated.

Not seeing pkgdown workflow in the Actions tab so that looks to be deleted.

**Comment by dicook** [comment]
Maybe its good!
